fsd_utils/config/commonconfig.py

```diff
@@ -58,6 +58,7 @@ class CommonConfig:
     # ---------------
 
     ASSESSMENT_STORE_API_HOST = os.getenv("ASSESSMENT_STORE_API_HOST", TEST_ASSESSMENT_STORE_API_HOST)
+    ASSESSMENT_STORE_API_HOST_DEV = "https://funding-service-design-assessment-store-dev.london.cloudapps.digital"
```

Comment:
I don't think this needs to be separate variable, can we just in-line it above?

Reply:
They aren't the same variable
`TEST_ASSESSMENT_STORE_API_HOST`
`ASSESSMENT_STORE_API_HOST_DEV`

Reply:
Why do we need a specific config var for the DEV host, surely, where this is needed it could override the standard host?